lego/apps/users/migrations/0039_remove_penalty_reason_remove_penalty_source_event_and_more.py

```diff
@@ -0,0 +1,144 @@
+# Generated by Django 4.0.10 on 2023-09-28 13:55
+
+import django.db.models.deletion
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("events", "0036_alter_registration_presence"),
+        ("users", "0038_alter_abakusgroup_type"),
+    ]
+
+    def generate_new_penalties(apps, schema_editor):
+        from django.utils import timezone
+        from django.utils.timezone import timedelta
+
+        PenaltyGroup = apps.get_model("users", "PenaltyGroup")
+        Penalty = apps.get_model("users", "Penalty")
+        Event = apps.get_model("events", "Event")
+
+        users_with_penalties = Penalty.objects.filter(
+            created_at__gt=timezone.now() - timedelta(days=10)
+        ).values_list("user", flat=True)
+
+        Penalty.objects.all().delete()
+
+        for user in users_with_penalties:
+            dummy_event = Event.objects.all()[0]
+            PenaltyGroup.objects.create(
+                user=user,
+                reason="Dette er en sammensetting av forrige prikkene",
+                source_event=dummy_event,
```

Comment:
I think that the source event here should be one of the events from the earlier penalties, and that the reason should include a list of the reasons that the user got the original penalties. Something like: "Dette er en sammensettning av tidligere prikker pga. migrering til nytt prikksystem. Prikkene som inngår i denne prikken er: 'kom for sent' (Julebord), 'slo til bedrep' (bedpress ned Warehouse), ..."